addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties

```diff
@@ -0,0 +1,140 @@
+# binding
+binding.netatmo.name = Netatmo Binding
+binding.netatmo.description = Dieses Binding integriert die smarte Wetterstation, die Zusatzmodule wie Regenmesser und Windmesser, als auch das smarte Thermostat.
+
+# bridge types
+thing-type.netatmo.netatmoapi.label = Netatmo API
+thing-type.netatmo.netatmoapi.description = Netatmo API.
+
+# bridge types config
+thing-type.config.netatmo.bridge.clientId.label = Client id
+thing-type.config.netatmo.bridge.clientId.description = Client id der APP, die unter https://dev.netatmo.com/myaccount/createanapp erstellt wurde.
+
+thing-type.config.netatmo.bridge.clientSecret.label = Client secret
+thing-type.config.netatmo.bridge.clientSecret.description = Client secret der APP, die unter https://dev.netatmo.com/myaccount/createanapp erstellt wurde.
+
+thing-type.config.netatmo.bridge.username.label = Benutzer
+thing-type.config.netatmo.bridge.username.description = Benutzer zur Authentifizierung an der Netatmo API.
+
+thing-type.config.netatmo.bridge.password.label = Passwort
+thing-type.config.netatmo.bridge.password.description = Passwort zur Authentifizierung an der Netatmo API.
+
+thing-type.config.netatmo.bridge.readStation.label = Wetterstation API
+thing-type.config.netatmo.bridge.readStation.description = Aktiviert den Zugriff auf die Wetterstation API (lesend).
+
+thing-type.config.netatmo.bridge.readThermostat.label = Thermostat API
+thing-type.config.netatmo.bridge.readThermostat.description = Aktiviert den Zugriff auf die Thermostat API (lesend und schreibend).
+
+# thing types
+thing-type.netatmo.NAMain.label = Haupt-Indoor-Modul
+thing-type.netatmo.NAMain.description = Haupt-Indoor-Modul, liefert Daten, wie z.B. Temperatur, Luftdruck, Luftfeuchtigkeit, CO2 Gehalt und Lautst�rke.
```

Comment:
@CWempe Thank you very much for taking the time.

I will change this.

@martinmv Do you know if the i18n supports  e.g. subscripted numbers? Maybe we could rename it to "CO₂-Gehalt".